templates/digital-asset-template/frontend/pages/CreateCollection.tsx

```diff
@@ -328,19 +348,29 @@ export function CreateCollection() {
             />
           </div>
           <div className="mb-5 flex flex-col item-center space-y-4">
-            <Label>Mint fee per NFT in APT (optional)</Label>
+            <Label
+              htmlFor="mint-fee"
+              tooltip="The fee the nft minter is paying the collection creator when they mint a NFT">
```

Comment:
```suggestion
              tooltip="The fee the nft minter is paying the collection creator when they mint an NFT">
```

Reply:
Not sure about the standards around this one. Is it common for `an` to be used with NFT?

Reply:
i think so, google said use a for non fungible token and an for NFT